CommonWallet/Classes/Common/Views/Assets/AccountListViewModelFactory.swift

```diff
@@ -0,0 +1,34 @@
+/**
+* Copyright Soramitsu Co., Ltd. All Rights Reserved.
+* SPDX-License-Identifier: GPL-3.0
+*/
+
+import Foundation
+import IrohaCommunication
+
+public protocol AccountListViewModelFactoryProtocol: class {
+    func createAssetViewModel(for asset: WalletAsset,
+                              balance: BalanceData,
+                              commandFactory: WalletCommandFactoryProtocol) -> AssetViewModelProtocol?
+    func createActionsViewModel(for assetId: IRAssetId?,
+                                commandFactory: WalletCommandFactoryProtocol) -> ActionsViewModelProtocol?
+    func createShowMoreViewModel(for delegate: ShowMoreViewModelDelegate?) -> WalletViewModelProtocol?
+}
+
+extension AccountListViewModelFactoryProtocol {
+    func createAssetViewModel(for asset: WalletAsset,
+                              balance: BalanceData,
+                              commandFactory: WalletCommandFactoryProtocol)
+        -> AssetViewModelProtocol? {
+        return nil
```

Comment:
Why nil?

Reply:
I refactored the logic of view models injection from the client a bit. Previously a client could provide a block which instantiates a viewModel on call. I replaced this with separate protocol which by default doesn't override any viewModel we have on dashboard returning nil.